src/App.js

```diff
@@ -1,10 +1,14 @@
 import React from "react";
 
+import Homepage from './components/homepage/homepage.component'
+
+
+
+
 const App = () => {
   return (
     <>
```

Comment:
I like your use of fragments. I'm wondering if Homepage could be used as app on this?